definitions/npm/react-redux_v4.x.x/flow_>=v0.24.x/react-redux_v4.x.x.js

```diff
@@ -0,0 +1,33 @@
+type ReactReduxConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function>(
+  mapStateToProps: (state: Object) => SP,
+  mapDispatchToProps: (dispatch: Dispatch) => DP,
```

Comment:
Also, `mapStateToProps` can accept a `function` with `Object` return type. There's an example in the `reselect` README.